CHANGELOG.md

```diff
@@ -1,14 +1,23 @@
 # Changelog
 
+## 2.24.0 - 2021-08-13
+#### New
+- New output: **Grafana** ([PR#254](https://github.com/falcosecurity/falcosidekick/pull/254))
+- New output: **Fission** ([PR#255](https://github.com/falcosecurity/falcosidekick/pull/254) thanks to [@gauravgahlot](https://github.com/gauravgahlot))
```

Comment:
I think there is a wrong reference here, it should be https://github.com/falcosecurity/falcosidekick/pull/255.